10up/insert-special-characters
# feat/34: Add faux cursor to preserve context

### Description of the Change



Closes #34 

### How to test the Change


1. Add a post with some content.
2. Click the insert special characters icon and observe the faux cursor.

### Video

**Faux cursor**

https://github.com/10up/insert-special-characters/assets/17757960/a4d6c64e-5289-4da4-801f-e572bf2a20fd

**Faux text selection**

https://github.com/10up/insert-special-characters/assets/17757960/a1190daa-72c1-4101-a9d3-b93d58194b51


### Changelog Entry

> Added - Faux cursor to preserve context

### Credits

Props @helen @Sidsector9 


### Checklist:
